api/provider/aws/task_read.go

```diff
@@ -0,0 +1,11 @@
+package aws
+
+import (
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (t *TaskProvider) Read(taskID string) (*models.Task, error) {
+	fqEnvironmentID := addLayer0Prefix(t.Config.Instance(), taskID)
```

Comment:
you cannot fetch the environment id from the task id. This will probably be done via tag lookup